src/debug_assert.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * This file and its contents are licensed under the Apache License 2.0.
+ * Please see the included NOTICE for copyright information and
+ * LICENSE-APACHE for a copy of the license.
+ */
+#ifndef TIMESCALEDB_DEBUG_ASSERT_H
+#define TIMESCALEDB_DEBUG_ASSERT_H
+
+#include <c.h>
+#include <postgres.h>
+
+/*
+ * Macro that expands to an assert in debug builds and to an ereport in
+ * release builds.
+ */
+#if DEBUG
+#define AssertOr(LEVEL, COND) Assert(COND)
```

Comment:
The name `AssertOr` confuses me. Sounds like there some `OR` condition as part of this assertion. What does the `Or` stand for? Do you mean `AssertOrError`, `AssertOrElog` or something?